Modules/Components/PreliminaryAffineStrategy/include/selxModulePreliminaryAffineStrategy.h

```diff
@@ -0,0 +1,37 @@
+/*=========================================================================
+*
+*  This Preliminary affine Strategy Module developed by Rupert Brooks in the context
+*  of the Continuous Registration Challenge.  June 2018
+*
+*  Copyright Leiden University Medical Center, Erasmus University Medical
+*  Center and contributors
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*        http://www.apache.org/licenses/LICENSE-2.0.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+*=========================================================================*/
+
+#include "selxTypeList.h"
+#include "selxPreliminaryAffineStrategyRegistrationComponent.h"
+namespace selx
+{
+using ModulePreliminaryAffineStrategyComponents = selx::TypeList<
+	//PreliminaryAffineStrategyRegistrationComponent< 2, unsigned char>,
```

Comment:
I had to comment these - something funny is going on when the input pixel type is not float.

Reply:
Perhaps some of the components that this module connects to also need to be compiled for float?